week2/homework/exercises/1-blog-API/server.js

```diff
@@ -1,10 +1,119 @@
-const express = require('express')
+const express = require('express');
 const app = express();
- 
-
-// YOUR CODE GOES IN HERE
-app.get('/', function (req, res) {
-  res.send('Hello World')
-})
- 
-app.listen(3000)
\ No newline at end of file
+const fs = require('fs');
+const path = require('path');
+app.use(express.json());
+
+app.post(   '/blogs',        (req, res) => { createBlog(req, res) });
```

Comment:
```suggestion
app.post(   '/blogs',  createBlog);
```
allso works because the parameters are the same

Reply:
Wow, nice to know! Thank you! 👍🏽